addons/storyshots/package.json

```diff
@@ -37,6 +37,7 @@
   },
   "peerDependencies": {
     "@storybook/addons": "4.0.0-alpha.1",
+    "@storybook/core": "4.0.0-alpha.1",
```

Comment:
We shouldn't add it as peerDependency unless users are supposed to install it on their side